knife/Gemfile.lock

```diff
@@ -1,35 +1,546 @@
+GIT
```

Comment:
@tpowell-progress Specific reason to add all the dependencies under knife/Gemfile.lock? This will unnecessarily pin versions for knife gem which is not needed.
The only reason we even created the knife/Gemfile.lock was to let bundler know there's separate chef ucrt gem when knife is being installed on windows systems. Hence kept only chef gem dependencies. Without it knife gem install on windows machine pulls in chef.gem instead of ucrt version of it.

Reply:
Removed the update, since Inspec-core is not mentioned.